README.md

```diff
@@ -44,16 +44,18 @@ These tools are required to work on the Assistant locally.
 
 ## Testing the Assistant
 
-Sketch isn't required to test the Assistant and run the [Jest](https://jestjs.io/) tests. So using a
-TDD approach together with real Sketch files as fixtures represents the fastest route to running
-your Assistant code and checking that it's working as expected.
+The Sketch Mac app isn't required to test the Assistant and run the [Jest](https://jestjs.io/)
+tests. So a TDD approach using Sketch files as fixtures represents the fastest route to running your
+Assistant and checking that it's working as expected.
 
 The tests can be run in watch mode for live feedback as you work.
 
 ```
 yarn test --watch
 ```
 
+### Test Github Action
```

Comment:
It reads a bit oddly, as if I'm testing the GitHub action, maybe something like this:

```suggestion
###  Github Action
```

```suggestion
### Using Github Actions
```

```suggestion
### Testing using Github Actions
```